tools/docs/config.ts

```diff
@@ -175,6 +209,16 @@ export async function generateConfig(dist: string, bot = false): Promise<void> {
       configOptionsRaw[headerIndex] +=
         `\n${option.description}\n\n` +
         genTable(Object.entries(el), option.type, option.default);
+
+      if (el.experimental) {
+        const footerIndex = configOptionsRaw.indexOf(
```

Comment:
🤔 I'm thinking if we should use a markdown parser or create an index of all sections, so it's more easy to add the generated data. 🤔

Reply:
if we do index it before we add the auto generated content wont it get corrupted once we add it? 
at the very least we will have to do some housekeeping

Reply:
ive indexed the markdown files. seems cleaner.
here's the only diff for this PR in the generated files - 
![image](https://user-images.githubusercontent.com/97394622/175200665-237d8586-120a-4506-a7e2-6d677a5a3166.png)